scripts/audio-format-converter/script.py

```diff
@@ -0,0 +1,67 @@
+# Script to Convert audio files from one format to another
+import os
+from glob import glob
+import pydub
+import mutagen
+import keyboard
+
+# enter dir of audio file
+song_dir = input("Enter : audio file path\n>> ")
+Path_Exist = os.path.isfile(song_dir)
+if Path_Exist == False:
+    print("Invalid")
+    quit()
+
+
+def conversion(song):
+    for song in song:
+        # Extension in which file is supposed to be converted
+        mp3_song = os.path.splitext(song)[0] + ".wav"
+        """
+        Support .mp3,
+                .mp4,
+                .ogg,
+                .aac,
+                .flv,
+                .mkv,
+                .flac,
+                .wav
+        """
```

Comment:
great work !! Can you make this dynamic. like user can choose the format or you can also pass those format as an argument.

Reply:
I'm a bit busy as of now, due to my own college stuff. Will make changes over time!

Reply:
Yeah no problem !

Reply:
So i tried taking a str input for the conversion format and concatenate it. It has broken the script for some reason. Hence why I'd like to close this issue & work on something else. Will be contributing more :)

Reply:
yes we highly encourage you to contribute here with more PR !  so do you want me to close this PR or you are going to work on this ?